src/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_subdirectory(libtools)
 add_subdirectory(tools)
+add_subdirectory(tests)
```

Comment:
We could put the `if(ENABLE_TESTING)` around the `add_subdirectory(tests)`, then we don't have to do it in the `CMakeLists.txt` itself.